hack/bin/integration-cleanup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Script to delete any helm releases prefixed with `test-` older than 20 minutes deemed inactive
+#
+# Motivation: cleanup of old test clusters that were not deleted (e.g. by the CI system, because it broke)
+
+date_seconds() {
+  d=$1
+  if [[ $OSTYPE =~ darwin* ]]; then
+    date -j -f "%b %d %H:%M:%S %Y" "$date_str" +"%s"
+  else 
+    date -d "$d" +"%s"
+  fi
+}
+
+date_now=$(date +"%s")
+
+# Since helm only shows information as hard-to-parse text pending https://github.com/kubernetes/helm/pull/2950
+# we are stuck with brittle grep/awk magic.
+
+IFS=$'\n'
+for release in $(helm list -A | grep test-); do
+  name=$(echo "$release" | awk '{print $1}')
+  namespace=$(echo "$release" | awk '{print $2}')
+  date_str=$(echo "$release" | awk '{print $4" "$5" "$6}')
+  date_s=$(date_seconds "$date_str")
+  diff=$(( $date_now - $date_s ))
+  if [ $diff -ge 7200 ]; then
+    echo "test release '$name' older than 2 hours; deleting..."
+    helm delete -n "$namespace" "$name"
+  fi
+
+done
```

Comment:
This will delete developer releases too, if we start using the CI cluster (I don't have enough RAM to run HLS, Firefox and kubernetes :sweat_smile: ). Maybe we can rename the cluster names or make this less aggressive.
Anyway, not a big problem since this will probably only happen time to time.

Reply:
My experience is that there these long-lived developer namespaces then hanging around. This is for ephemeral testing only, not for long-lived things. If it gets cleaned up the next kube-integration takes an extra 3 minutes. I think this may be bearable. But we can increase the kubernetes cluster size so we can host 10-15 ephemeral environments at a time, and then make this less agressibe.